solr/solr-ref-guide/modules/indexing-guide/pages/reindexing.adoc

```diff
@@ -165,6 +165,20 @@ Once the alias is in place and you are satisfied you no longer need the old data
 One advantage of this option is that you can switch back to the old collection if you discover problems our testing did not uncover.
 Of course this option can require more resources until the old collection can be deleted.
 
+=== Exporting/Importing Data from Solr
+
+Sometimes you don't want to run your full ETL pipeline to reindex into another collection, you just want to take the data in your existing collection, export it, and then import it back.
```

Comment:
[-0] AFAIK `bin/solr export` uses the `/export` handler and [can only return fields that have docValues enabled](https://solr.apache.org/guide/solr/latest/query-guide/exporting-result-sets.html#field-requirements).  So it'd drop most text fields, etc.

That's a huge limitation that we should probably mention here!